data/swtpm/ssh_script

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/expect
+ 
+# This script is used for access remote host via ssh and run some commands
+# without interactive operations
+# Usage: ssh_script [host] [user] [password]
+
+
+set a_exp $::env(MY_VARIABLE)
```

Comment:
`MY_VARIABLE` isn't really a suitable variable name.

Reply:
let me fix it.